package.json

```diff
@@ -17,8 +17,17 @@
     "type": "git",
     "url": "git+https://github.com/mhlabs/dynamodb-client.git"
   },
+  "scripts": {
+    "test": "vitest ./src",
+    "integration-tests": "vitest ./integration-tests"
```

Comment:
Minor thing, but these days we usually use delimiters instead, so this would be `test:integration`, not sure how you do in your team but I prefer delimiters. Not a big deal though..